orchestrator/careplanservice/handle_getcareplan.go

```diff
@@ -47,6 +48,12 @@ func (s *Service) handleSearchCarePlan(ctx context.Context, queryParams url.Valu
 	if err != nil {
 		return nil, err
 	}
+
+	if len(bundle.Entry) == 0 {
+		log.Info().Msg("CarePlan search returned empty bundle")
```

Comment:
Would this occur commonly? I think not something you want to log on INFO (if you want to log it at all)

Reply:
Hmm that's a fair point. I picked it up happening because a search for CarePlan would be done before the Task was created (I think for the CarePlan selection)